src/admin-views/tribe-options-tickets.php

```diff
@@ -93,12 +93,21 @@
 			'parent_option'   => Tribe__Events__Main::OPTIONNAME,
 			'default'         => reset( $ticket_form_location_options ),
 		),
+	) );
+}
+
+if ( class_exists( 'Tribe__Tickets_Plus__Main' ) ) {
```

Comment:
I think using the filter `tribe_tickets_settings_tab_fields` instead on Tickets+ would remove the need of the class `class_exists` as will be created directly on event Tickets+, and also favors extensibility.

By making sure this item is modified only where is required by the item that requires this change.

Let me know